src/main.rs

```diff
@@ -405,16 +405,10 @@ fn run() -> Result<()> {
         print_info("\n(R)eboot\n(S)hell\n(Q)uit");
         loop {
             match get_key() {
-                Ok(Key::Char('s')) | Ok(Key::Char('S')) => {
-                    run_shell();
-                }
-                Ok(Key::Char('r')) | Ok(Key::Char('R')) => {
-                    reboot();
-                }
-                Ok(Key::Char('q')) | Ok(Key::Char('Q')) => (),
-                _ => {
-                    continue;
-                }
+                Ok(Key::Char('s' | 'S')) => run_shell(),
```

Comment:
This is nice. Could you tell me which feature of Rust are you using here?

Reply:
Nested "or patterns", introduced in Rust 1.53